guide/using-the-pioneer-3-at-robot.md

```diff
@@ -25,6 +25,13 @@ More information on the specifications and optional devices is available on Adep
 %end
 
 The standard model of the Pioneer 3-AT is provided in the "pioneer3AT.wbt" file which is located in the "WEBOTS\_HOME/projects/robots/adept/pioneer3/worlds" directory of the Webots distribution.
+
+%figure "Pioneer 3-AT world example"
+
+![pioneer3at_example.png](images/pioneer3at_example.png)
+
+%end
+
```

Comment:
A figure should always be referred to from the text. So you may add some "(see figure xxx)" in the above text or the like.